gradle/dependencies.gradle

```diff
@@ -13,7 +13,7 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '123.0.0'
+      mapboxNavigatorVersion = '123.1.0'
```

Comment:
No changelog entries are required?

Reply:
All changes seem internal. I'll double check with the team.